evap/staff/fixtures/excel_files_test_data.py

```diff
@@ -0,0 +1,132 @@
+duplicate_user_import_filedata = {
+    "Users": [
```

Comment:
Just because I’m curious: Do you have a pattern when to use `'` and when `"`? I would prefer to keep them consistently `'`, but that’s not a strong opinion here

Reply:
Nope, I got no pattern 😅